servers/src/common/hooks.rs

```diff
@@ -262,9 +263,9 @@ impl WebHook {
 impl ChainEvents for WebHook {
 	fn on_block_accepted(&self, block: &core::Block, status: &BlockStatus) {
 		let status = match status {
-			BlockStatus::Reorg => "reorg",
-			BlockStatus::Fork => "fork",
-			BlockStatus::Next => "head",
+			BlockStatus::Reorg(depth) => format!("reorg:{}", depth),
```

Comment:
If we are building json here then might make sense to add a new (optional) field for the reorg depth.
It would save clients from having to parse this out of the string on the other side.

Reply:
```
# optional depth field, only used for reorg status
{
  "hash": "abc",
  "status: "reorg",
  "depth": 1,
  "data": { 
    ... 
  }
}

# maybe status is optionally a dict and not just a simple value (harder to parse though)
{
  "hash": "abc",
  "status: {
    "reorg": 1,
  },
  "data": { 
    ... 
  }
}
```

Reply:
@antiochp I've made option no. 1 that adds optional `depth` field on the top level.

Reply:
:+1: Looks good to me. 

We should get some feedback from someone who uses this endpoint.

Reply:
@kargakis 👀

Reply:
Looks good to me, thanks for working on this @eupn !